diff --git a/dist/pool/index.mjs b/dist/pool/index.mjs
index ba23b53f1af6a9aec52e367c06daf3da75d509dc..07ce0517fb34cc3763ac22ae69d0be9e517524e6 100644
--- a/dist/pool/index.mjs
+++ b/dist/pool/index.mjs
@@ -12604,7 +12604,7 @@ async function handleModuleFallbackRequest(vite, request) {
 //#region src/pool/config.ts
 const PLUGIN_VALUES = Object.values(PLUGINS);
 const WorkersPoolOptionsSchema = z.object({
-	main: z.ostring(),
+	main: z.string().optional(),
 	remoteBindings: z.boolean().default(true),
 	additionalExports: z.record(z.string(), z.union([
 		z.literal("WorkerEntrypoint"),
@@ -12613,8 +12613,8 @@ const WorkersPoolOptionsSchema = z.object({
 	])).default({}),
 	miniflare: z.object({ workers: z.array(z.object({}).passthrough()).optional() }).passthrough().optional(),
 	wrangler: z.object({
-		configPath: z.ostring(),
-		environment: z.ostring()
+		configPath: z.string().optional(),
+		environment: z.string().optional()
 	}).optional()
 });
 function isZodErrorLike(value) {
